docs/index.md

```diff
@@ -1,1241 +1,21 @@
-A framework for building reusable, testable & encapsulated view components in Ruby on Rails.
+---
+layout: default
+title: Overview
+nav_order: 1
+---
+
+# ViewComponent
 
-[View on GitHub →](https://github.com/github/view_component)
+A framework for building reusable, testable & encapsulated view components in Ruby on Rails.
```

Comment:
As-is this page is a little light on content but seems like the best place to "sell" the idea of view components to visitors.

I don't think it's blocking, but it could be cool to show off some of the features of view components. Maybe a good example could be comparing helpers/partials to a component that implements slots and some other functionality?